src/ducible/patch_image.cpp

```diff
@@ -124,6 +124,11 @@ void patchDebugDataDirectories(const PEFile& pe, Patches& patches,
     size_t debugDirCount;
     auto dir = pe.getDebugDataDirs(opt, debugDirCount);
 
+    if( ! dir ) {
+      // No debug directories (MinGW executable?)
+      return;
+    }
+
```

Comment:
This should be changed to just
```cpp
if (!dir) return;
```

The style should match the rest of the code.

I also noticed that this fixes a crash when running this on .NET assemblies.